WooCommerce/Classes/POS/Presentation/Reusable Views/POSConnectivityView.swift

```diff
@@ -28,19 +28,19 @@ struct POSConnectivityView: View {
     @ViewBuilder private var noConnectionBanner: some View {
         HStack(spacing: Constants.spacing) {
             Image(systemName: "wifi.exclamationmark")
-                .foregroundColor(Color(.text.inverted))
+                .foregroundColor(Color.posOnSecondaryContainer)
                 .font(.posBodySmallBold)
 
             Text(Localization.title)
-                .foregroundColor(Color(.text.inverted))
+                .foregroundColor(Color.posOnSecondaryContainer)
                 .font(.posBodySmallBold)
         }
         .padding(.vertical, Constants.verticalPadding)
         .padding(.horizontal, Constants.horizontalPadding)
         .frame(minHeight: Constants.height)
-        .background(Color(.systemGray6.inverted))
+        .background(Color.posSecondaryContainer)
         .cornerRadius(Constants.cornerRadius)
-        .shadow(color: Color.black.opacity(0.2), radius: 8, x: 0, y: 2)
+        .shadow(color: Color.posShadow.opacity(0.2), radius: 8, x: 0, y: 2)
```

Comment:
🗒️ Note for reviewer: while reviewing this view, I noticed some color usage not in the design system and thought it saves some time updating in an existing PR. Replacing these colors also meant that `Color+Inverted` can be deleted. Please let me know if you prefer having these in a separate PR. Preview in both color schemes:

<img width="945" alt="Screenshot 2025-02-13 at 1 48 56 PM" src="https://github.com/user-attachments/assets/b6555cc4-316f-442e-844c-83796df605be" />

Reply:
Totally fine to have it here, thanks for addressing it!